src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -113,16 +113,18 @@ function insertUpdateIntoFiber<State>(
   const alternateFiber = fiber.alternate;
   let queue1 = fiber.updateQueue;
   if (queue1 === null) {
-    queue1 = fiber.updateQueue = createUpdateQueue(fiber.memoizedState);
+    // TODO: We don't know what the base state will be until we begin work.
+    // It depends on which fiber is the next current. Initialize with an empty
+    // base state, then set to the memoizedState when rendering. Not super
```

Comment:
If we just remove the argument to `createUpdateQueue` would it be that bad?

Reply:
@gaearon I'd prefer to keep it for type safety for queues that don't belong to Fiber. Don't have any of those yet but could in the future (I do in one of my PRs).

Reply:
OK